docs/rules/accessibility/no-aria-label-misuse.md

```diff
@@ -0,0 +1,45 @@
+# No aria label misuse
+
+## Rule Details
+
+This rule aims to minimize misuse of the `aria-label` and `aria-labelledby` attributes because the usage of these attributes is only guaranteed on interactive elements and a subset of ARIA roles. WSC provides [a list of ARIA roles which cannot be named](https://w3c.github.io/aria/#namefromprohibited) which is used as a basis for this linter.
```

Comment:
I think this was a typo; if not, can we please spell out “WSC”?
```suggestion
This rule aims to minimize misuse of the `aria-label` and `aria-labelledby` attributes because the usage of these attributes is only guaranteed on interactive elements and a subset of ARIA roles. W3C provides [a list of ARIA roles which cannot be named](https://w3c.github.io/aria/#namefromprohibited) which is used as a basis for this linter.
```